packages/fx-core/templates/plugins/resource/keyvault/bicep/keyVaultProvision.template.bicep

```diff
@@ -0,0 +1,54 @@
+@secure()
+param provisionParameters object
+param userAssignedIdentityObjectId string
+
+var resourceBaseName = provisionParameters.resourceBaseName
+var keyVaultName = contains(provisionParameters, 'keyVaultName') ? provisionParameters['keyVaultName'] : '${resourceBaseName}'
+var tenantId = subscription().tenantId
+var m365ClientSecretName = 'm365ClientSecret'
+var m365ClientSecret = provisionParameters['m365ClientSecret']
+var botClientSecretName = 'botClientSecret'
+var botClientSecret = contains(provisionParameters, 'botAadAppClientSecret') ? provisionParameters['botAadAppClientSecret'] : ''
+
+
+resource keyVault 'Microsoft.KeyVault/vaults@2019-09-01' = {
+  name: keyVaultName
+  location: resourceGroup().location
+  properties: {
+    tenantId: tenantId
+    accessPolicies: [
+      {
+        tenantId: tenantId
+        objectId: userAssignedIdentityObjectId
+        permissions: {
+          secrets: [
+            'get'
+          ]
+        }
+      }
+    ]
+    sku: {
+      name: 'standard'
```

Comment:
also expose parameters for SKU, and add a comment telling user how to customize SKU via parameters. You can refer existing bicep files as example.
Is it enough to only include `name` property in SKU, or we must provide the `family` property?

Reply:
@MuyangAmigo Please help advice.
`Azure Key Vault has two service tiers: Standard, which encrypts with a software key, and a Premium tier, which includes hardware security module(HSM)-protected keys.`
In our scenario, we do not need `Premium tier`.